lib/util/init.config.js

```diff
@@ -239,13 +251,49 @@ const Init = {
             propertiersCur[fieldName] = defaultPropsCur[fieldName];
         }
     },
+    /**
+     * returns list of files that need to be updated
+     * @param {string} projectVersion version found in config file of the current project
+     * @returns {string[]} relevant files with path that need to be updated
+     */
+    _getForcedUpdateList(projectVersion) {
+        // list of files that absolutely need to get overwritten, no questions asked, when upgrading from a version lower than the given.
+        // !newest entry on top, sorted by version number!
+        let forceIdeConfigUpdate;
+        if (File.existsSync(Util.configFileName)) {
+            try {
+                forceIdeConfigUpdate = File.readJsonSync(
+                    path.resolve(__dirname, Util.boilerplateDirectory, 'forcedUpdates.json')
+                );
+            } catch (ex) {
+                Util.logger.error(`${ex.code}: ${ex.message}`);
+                return [];
```

Comment:
is there a scenario where this wouldnt exist? if not, then lets throw a proper error and fail.